osquery/tables/system/windows/security_profile_info.cpp

```diff
@@ -0,0 +1,495 @@
+/**
+ * Copyright (c) 2014-present, The osquery authors
+ *
+ * This source code is licensed as defined by the LICENSE file found in the
+ * root directory of this source tree.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+ */
+
+#include <boost/noncopyable.hpp>
+
+#include <osquery/core/core.h>
+#include <osquery/core/tables.h>
+#include <osquery/logger/logger.h>
+#include <osquery/utils/conversions/windows/strings.h>
+
+namespace osquery {
+namespace tables {
+
+/**
+ * @brief Manages access to SCE RPC client API exported in scecli.dll
+ */
+class SceClientHelper : public boost::noncopyable {
```

Comment:
Ah, so I think we tend to bring the decls into their own header file, and leave just the implementation code in the cpp.

Reply:
Also you'll want to move the impl of the class out into it's own source file as well, so you'd have 1 source file for table data generation, a header file for the RIAA management, and lastly the impl file for the class.

Reply:
Hey @muffins, thanks for looking into this! I've made the suggested code refactor and split the utils from the table generation logic. I agree this is better for keeping code cleaner and more purposeful.